src/dictionary.c

```diff
@@ -182,7 +183,7 @@ uint32_t dict_key_size(SEXP x) {
     r_stop_internal("Dictionary functions do not support long vectors.");
   }
 
-  const double load_adjusted_size = x_size / 0.77;
+  const double load_adjusted_size = x_size / 0.50;
```

Comment:
Maybe fetch the load factor from an envvar at load time for easier experimentation?

Reply:
oh I guess a `"vctrs:::"` global option makes more sense

Reply:
I think it is reasonable to do this if it comes up again. But for now I don't feel the need to add in the little bit of extra complexity for something that I am hopeful won't need to change again